docs/NPCs/Red Wizards/Shadowmantle (aka Valindra Cimber).md

```diff
@@ -1,5 +1,11 @@
-[[Xandala Cimber]]s sister, was a soldier in a war. Does not want anything to do with [[Xandala Cimber]], and they haven't seen each other in six years.
+[[Xandala Cimber]]'s sister, was a soldier in a war. Does not want anything to do with [[Xandala Cimber]], and they haven't seen each other in six years.
 
 She likes frogs, and turned out to be Shadowmantle!
 
+After her father left, she took care of the house on her own. Eventually got drafted into the Cormyrian war wizard army fighting Netheril.
+
+She rose through the ranks of [[The Red Wizards of Thay]] quicker than just about anyone else. She's only been part of them for the past three or so years, but she's nearly at the top of their ranks.
```

Comment:
She was part of the red wizards for at least 3.5 years, but probably joined closely after the war ended